### ISSUE TITLE
Missing `*Sync` synchronous functions

### ISSUE THREAD

**Issue by EthanThatOneKid** [cross-org/fs#1]
Is it within the scope of this project to expose the synchronous versions of the fs functions?

**Comment by Hexagon** [comment]
I see the *Sync as a legacy from the past, to simplify callback/promise hell before async/await arrived etc. But If there are good use cases, we should of course include them :+1:

**Comment by EthanThatOneKid** [comment]
Accessing these functions in synchronous contexts can still be convenient in some situations.

Whether *Sync becomes part of `@cross/fs` or not, I'm curious if we can achieve the same cross-runtime support on jsr.io. Would importing from `node:fs` suffice?

**Comment by Hexagon** [comment]
@EthanThatOneKid Yep, node:fs seems to be working fine in all of Node, Deno and Bun 👍

**Comment by Pinta365** [comment]
We’re keeping @cross/fs async-only. For sync APIs, node:fs Sync (e.g. readFileSync) works in Node, Deno, and Bun, so you can import from node:fs when you need sync.
